deep-learning/src/test/python/synapsemltest/dl/test_deep_text_classifier.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+from pyspark.ml.evaluation import MulticlassClassificationEvaluator
+from pyspark.sql import SparkSession
+from pytorch_lightning.callbacks import ModelCheckpoint
+from synapse.ml.dl import *
+
+import pytest
+from .conftest import CallbackBackend, _prepare_text_data
+from .test_deep_vision_classifier import local_store
+from .test_deep_vision_model import MyDummyCallback
+
+
+@pytest.mark.skip(reason="not testing this for now")
+def test_bert_base_cased():
```

Comment:
I see this is using pytest, but all of the autogenned tests are in unittest style. We also already have functionality that creates a spark session (it just hgappens at test generation time). Perhaps we can align these tests with the existing test style. But let me know if you think this will pose challenges or problems